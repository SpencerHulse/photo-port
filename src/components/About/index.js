import coverImage from "../../assets/cover/cover-image.jpg";

function About() {
  return (
    <section className="my-5">
      <h1 id="about">Who am I?</h1>
      <img
        src={coverImage}
        className="my-2"
        style={{ width: "100%" }}
        alt="cover"
      />
      <div>
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. At quos
        voluptatibus, saepe culpa soluta fuga, unde autem ipsa accusamus
        nesciunt amet et illo eos, distinctio sed! Qui atque minima quis? Lorem
        ipsum dolor sit, amet consectetur adipisicing elit. Molestias quibusdam
        nobis voluptatem debitis repudiandae. Expedita, culpa, quisquam vero
        rerum id cumque esse nisi a labore ut eius totam ab maxime?Lorem ipsum
        dolor sit amet consectetur, adipisicing elit. Impedit eum sit
        necessitatibus error ipsa aperiam magni laborum optio laboriosam quis,
        natus iste. Commodi quasi nobis nesciunt iure assumenda cupiditate a!
      </div>
    </section>
  );
}

export default About;
