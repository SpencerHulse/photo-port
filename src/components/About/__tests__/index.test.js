import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import About from "..";

afterEach(cleanup);

describe("About component", () => {
  // First test - verify that the component is rendering
  test("renders", () => {
    render(<About />);
  });

  // Second test - test case - comparing snapshots
  test("matches snapshot DOM node structure", () => {
    // Render About
    const { asFragment } = render(<About />);
    expect(asFragment()).toMatchSnapshot();
  });
});
