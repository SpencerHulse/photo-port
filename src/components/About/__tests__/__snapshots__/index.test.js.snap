// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <h1
      id="about"
    >
      Who am I?
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="cover-image.jpg"
      style="width: 100%;"
    />
    <div>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. At quos voluptatibus, saepe culpa soluta fuga, unde autem ipsa accusamus nesciunt amet et illo eos, distinctio sed! Qui atque minima quis? Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias quibusdam nobis voluptatem debitis repudiandae. Expedita, culpa, quisquam vero rerum id cumque esse nisi a labore ut eius totam ab maxime?Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit eum sit necessitatibus error ipsa aperiam magni laborum optio laboriosam quis, natus iste. Commodi quasi nobis nesciunt iure assumenda cupiditate a!
    </div>
  </section>
</DocumentFragment>
`;
